man/repart.d.xml

```diff
@@ -831,6 +845,9 @@ VerityMatchKey=root
 Type=root-verity
 Verity=hash
 VerityMatchKey=root
+# Explicitly set the hash and data block size to 4K. The default for both is the sector size.
```

Comment:
This isn't entirely correct anymore, we now use 4k as the default if we're not operating on a block device and no sector size is explicitly configured.

Reply:
"Configures the hash block size of the generated verity hash partition. Defaults to 4K for an image, or the underlying block device sector size. Must be between 512 and 4096 bytes and must be a power of 2."

Adjust as required and I'll get it fixed up.

Reply:
I would just drop the mention of the default value here. So keep the first sentence and drop the second one

Reply:
I updated the other portion of the documentation too. Can you take a look there too and update the wording if it needs to be done?